### PR TITLE
Feature/performance

### PR DESCRIPTION
**Pull request by ClintonYounge**
### UPDATES AND ADDED FEATURES
- Created an email column in the owners table.
- Add values to visits table and email column.
- Created indexes.
-  Added three performance query tests.

### Screenshots
**First Query Before and after creating index**
performance ss1 before
*From 1058ms TO 269ms*
performance ss1 after

**Second Query Before and after creating index**
performance ss2 before
*From 3958ms TO 1748ms*
performance ss2 after

**Third Query Before and after creating index**
performance ss3 before
*From 2345ms TO 0.08ms*
performance ss3 after

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -179,4 +179,11 @@ JOIN species ON animals.species_id = species.id
...
-FETCH FIRST 1 ROW ONLY;
\ No newline at end of file
+FETCH FIRST 1 ROW ONLY;
+
+
+-- Performance
+
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYSE SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYSE SELECT * FROM owners where email = 'owner_18327@mail.com';
```
Comment (Gambit142) [inline queries.sql:189]: - [x] Based on the requirements, the execution time for these SQL queries should be `>= 1000ms`. However, I noticed that the execution time for this query is below `1000ms`:

```SQL
EXPLAIN ANALYSE SELECT * FROM owners where email = 'owner_18327@mail.com';
```
I would advise you to run this query two or three more times until the execution time is `>= 1000ms`. 

```SQL
insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

#### Hint: Kindly refer to this [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md) 👍 😎

File: queries.sql [hunk 2]
```diff
@@ -179,4 +179,11 @@ JOIN species ON animals.species_id = species.id
...
 ORDER BY COUNT(species.name) DESC
-FETCH FIRST 1 ROW ONLY;
\ No newline at end of file
+FETCH FIRST 1 ROW ONLY;
+
+
+-- Performance
+
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYSE SELECT * FROM visits where vet_id = 2;
```
Comment (Gambit142) [inline queries.sql:188]: - [OPTIONAL] Even if the execution times difference is very small for this query:

```SQL
EXPLAIN ANALYSE SELECT * FROM visits where vet_id = 2;
```
In your case, I can clearly see that you had some fun with the `explain analyze` SQL statement and I am sure that in the future you will know how to start the investigation for some slow DB queries 💻 🖱️

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @ClintonYounge @MoyasiGinko 👋, 

Good job so far!

## Highlights
- Populated database with data ✔️ 
- Created index on required columns ✔️ 
- Use of the `explain analyze` command ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

#### You can also consider the following the comment tagged [OPTIONAL] under this review summary 👍 💻 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql (hunk 1 above)
Comment (ClintonYounge) [reply queries.sql:189]: Thanks a lot!

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @ClintonYounge and @MoyasiGinko 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**